src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_moved_to_error_queue_using_dtc.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_message_is_moved_to_error_queue_using_dtc : NServiceBusAcceptanceTest
```

Comment:
I'm all for leaving the second test outside of scope. By `would be tricky` I meant:
 * I think in practice only `SqlTransport` and 'NHibernate` persister can do that
 * In that case they still use `TransactionScope` but do not escalate to DTC
 * I don't think `Core` is proper place for doing that.